wkdehdlr/CoinHelper/app/src/main/java/kr/schoolsharing/coinhelper/base/BaseFragment.kt

```diff
@@ -9,7 +9,9 @@ import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
 import androidx.fragment.app.Fragment
 
-abstract class BaseFragment<B : ViewDataBinding>(@LayoutRes private val layoutId: Int) :
+abstract class BaseFragment<B : ViewDataBinding>(
+    @LayoutRes private val layoutId: Int
```

Comment:
어노테이션은 개행해주시는 것이 좋아요.
```suggestion
    @LayoutRes 
    private val layoutId: Int
```

Reply:
[b0f6acd](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/216/commits/b0f6acdf6805fa6f79a19256ba017ca1f3a19f14) 수정했습니다